portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,131 @@
+body{
+    margin: auto;
+    max-width: 1250px;
+    font-family: Impact, Charcoal, sans-serif;
+    font-size: 20px;
+}
+
+.container {
+    width: 100%;
+    padding: 30px;
+    max-width: 1200px;
+    margin: 0;
+}
+
+/* Horizontal alignment for picture and text on the front page */
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+    margin: auto;
+}
+
+#profile, #content {
+    float: left;
+    width: 47%;
+    margin: 0;
+    padding: 0;
+}
+
+#profile {
+    margin-right: 10px;
+    margin-top: 20px;
+    margin-left: auto;
+}
+
+/* page title styles */
+#title {
+    padding: 20px;
+}
+
+/* styles for divs for text on pages */
+.work-div, .education-div, .acomplishment-div, .project-div {
+    padding: 25px;
+}
+
+/* Navigation menu style */
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    background-color: #F3B5A7;
```

Comment:
Nit: all colors in CSS should be lowercase.

`#F3B5A7` -> `#f3b5a7`

https://google.github.io/styleguide/htmlcssguide.html#Capitalization